doc/src/build/zk_login.md

```diff
@@ -180,11 +180,11 @@ The OAuth URL can be constructed with `$CLIENT_ID`, `$REDIRECT_URL` and `$Nonce`
 
 User salt is used when computing the zkLogin Sui address (see [definition](#address-definition)). There are several options for the application to maintain the user salt:
 1. Client Side:
-    - Request user input for the salt during wallet access, transferring the responsibility to the user, who must then remember it.
-    - Browser or Mobile Storage: Ensure proper workflows to prevent users from losing wallet access during device or browser changes. One approach is to email the salt during new wallet setup.
+   - Request user input for the salt during wallet access, transferring the responsibility to the user, who must then remember it.
```

Comment:
I think these docs should avoid the use of the term "wallet" unless they are describing the piece of software (e.g., browser plugin) that manages keys.